dask_expr/reductions.py

```diff
@@ -94,49 +165,94 @@ def _layer(self):
 
         return d
 
-    @property
-    def _meta(self):
-        meta = meta_nonempty(self.frame._meta)
-        meta = self.chunk(meta, **self.chunk_kwargs)
-        meta = self.combine([meta], **self.combine_kwargs)
-        meta = self.aggregate([meta], **self.aggregate_kwargs)
-        return make_meta(meta)
 
-    def _divisions(self):
-        return [None, None]
+##
+## Simple class reductions
+##
+
+
+class DataFrameReduction(MapReduce):
```

Comment:
Does this work on Series objects?  If so, we maybe want to avoid DataFrame in a name.